dask_cloudprovider/providers/digitalocean/tests/test_droplet.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from dask_cloudprovider.providers.digitalocean.droplet import DropletCluster
+from dask.distributed import Client
+import dask.array as da
+from distributed.core import Status
+
+
+@pytest.fixture
+async def gen_cluster():
+    async with DropletCluster(asynchronous=True) as cluster:
+        yield cluster
+
+
+@pytest.mark.asyncio
+async def test_init():
+    cluster = DropletCluster(asynchronous=True)
+    assert cluster.status == Status.created
```

Comment:
`async with`?

Reply:
I am intentionally not awaiting the cluster here